pkgs/development/python-modules/fairseq/default.nix

```diff
@@ -41,7 +42,8 @@ buildPythonPackage rec {
     hash = "sha256-XX/grU5ljQCwx33miGoFc/7Uj9fZDtmhm4Fz7L4U+Bc=";
   };
 
-  disabled = pythonOlder "3.7";
+  # https://github.com/facebookresearch/fairseq/pull/5062
```

Comment:
We can't just fetch the patch? Python 3.11 is the default right now.

Reply:
I'm hesitant to due to this comment: https://github.com/facebookresearch/fairseq/pull/5062#issuecomment-1550145316

Reply:
Hm, there's a new one.
I'll try it out. https://github.com/facebookresearch/fairseq/pull/5359

Reply:
This is a bit too indev to add to nixpkgs, maybe another time.
https://github.com/omry/omegaconf/issues/1136

Reply:
Looked into it more, maybe I was just tired but I got it working now.